package.json

```diff
@@ -36,79 +36,83 @@
   "dependencies": {
     "@bigcommerce/stencil-paper": "^2.0.2",
     "@bigcommerce/stencil-styles": "1.1.0",
-    "accept-language-parser": "^1.0.2",
-    "archiver": "^0.14.4",
+    "accept-language-parser": "^1.4.1",
+    "archiver": "^2.0.0",
     "async": "^2.4.0",
     "babel-eslint": "^7.1.1",
-    "boom": "^2.7.0",
+    "boom": "^5.2.0",
     "browser-sync": "git://github.com/mcampa/browser-sync.git#tunnelBug",
-    "cheerio": "^0.19.0",
+    "cheerio": "^1.0.0-rc.2",
     "code": "^4.0.0",
     "colors": "^1.0.3",
     "commander": "^2.7.1",
-    "confidence": "^1.0.0",
+    "confidence": "^3.0.2",
     "dateformat": "^2.0.0",
-    "decompress-zip": "^0.2.0",
-    "eslint": "^2.2.0",
-    "eslint-config-airbnb": "^6.0.2",
-    "front-matter": "^1.0.0",
-    "github": "^8.1.0",
-    "glob": "^5.0.14",
-    "glue": "^2.0.0",
-    "good": "^5.1.2",
-    "good-console": "^4.1.0",
+    "decompress-zip": "^0.3.0",
+    "eslint": "^4.5.0",
+    "eslint-config-airbnb": "^15.1.0",
+    "eslint-plugin-react": "^7.3.0",
+    "front-matter": "^2.1.2",
+    "github": "^9.3.1",
+    "glob": "^7.1.2",
+    "glue": "^4.2.0",
+    "good": "^7.2.0",
+    "good-console": "^6.4.0",
+    "h2o2": "^6.0.0",
     "handlebars": "^3.0.3",
-    "hapi": "^8.4.0",
-    "hoek": "^2.12.0",
-    "image-size": "^0.4.0",
-    "inquirer": "^0.8.2",
-    "install": "^0.1.8",
+    "hapi": "^16.5.2",
+    "hoek": "^4.2.0",
+    "image-size": "^0.6.1",
+    "inert": "^4.2.1",
+    "inquirer": "^3.2.2",
+    "install": "^0.10.1",
     "jsonlint": "^1.6.2",
-    "jsonschema": "^1.0.2",
-    "jspm": "^0.15.7",
+    "jsonschema": "^1.2.0",
+    "jspm": "^0.16.53",
     "jspm-git": "^0.6.0",
-    "lab": "^5.5.1",
+    "lab": "^14.2.0",
     "less": "^2.5.0",
-    "lodash": "^3.9.0",
-    "memory-cache": "^0.1.4",
-    "npm": "^2.7.5",
-    "recursive-readdir": "^1.3.0",
+    "lodash": "^4.17.4",
+    "memory-cache": "^0.2.0",
+    "npm": "^5.3.0",
     "progress": "^2.0.0",
+    "recursive-readdir": "^2.2.1",
     "request": "^2.81.0",
-    "semver": "^5.3.0",
-    "simple-git": "^1.65.0",
-    "sinon": "^2.2.0",
+    "semver": "^5.4.1",
+    "simple-git": "^1.75.0",
+    "sinon": "^3.2.1",
     "tarjan-graph": "^0.3.0",
-    "tmp": "0.0.26",
-    "upath": "^0.1.7",
+    "tmp": "0.0.33",
+    "upath": "^1.0.0",
     "uuid4": "^1.0.0",
+    "vision": "^4.1.1",
     "when": "^3.7.7",
-    "wreck": "^12.2.2"
+    "wreck": "^12.2.3"
   },
   "devDependencies": {
-    "angular": "1.5.6",
-    "angular-animate": "1.5.6",
-    "angular-cache": "3.2.4",
-    "angular-cookies": "1.5.6",
-    "angular-gettext": "1.1.4",
-    "angular-sanitize": "1.5.6",
-    "angular-ui-router": "0.2.13",
-    "babel-core": "^6.8.0",
-    "babel-loader": "^6.2.4",
-    "babel-preset-es2015": "^6.6.0",
+    "angular": "^1.6.6",
```

Comment:
You'll have to re-bundle theme-editor and test it with this version of angular and webpack